diff --git a/-private/environment/index.js b/-private/environment/index.js
index 217583f4bae53f6f72db8d3debd0f31b3dac2b8e..aa2c1a2a665005d8c95b9e3018640fdf7e317861 100644
--- a/-private/environment/index.js
+++ b/-private/environment/index.js
@@ -46,10 +46,6 @@ function emberLooseEnvironment(options) {
                     // Pod component/controller/route
                     candidates.push(templatePath.replace(REGEXES.POD_TEMPLATE, '/component.ts'), templatePath.replace(REGEXES.POD_TEMPLATE, '/controller.ts'), templatePath.replace(REGEXES.POD_TEMPLATE, '/route.ts'));
                 }
-                else if (templatePath.includes('/templates/components/')) {
-                    // Classic component
-                    candidates.push(colocatedTsScriptPath.replace('/templates/components/', '/components/'));
-                }
                 else if (templatePath.includes('/templates/')) {
                     // Classic controller/route
                     candidates.push(colocatedTsScriptPath.replace('/templates/', '/controllers/'), colocatedTsScriptPath.replace('/templates/', '/routes/'));
