diff --git a/src/app-files.js b/src/app-files.js
index 924a11241f9e6584fee5aba793e9cd86ea6d708f..6822e87bba314adeacee240964fd6f3317349088 100644
--- a/src/app-files.js
+++ b/src/app-files.js
@@ -74,10 +74,10 @@ class AppFiles {
                 }
                 continue;
             }
-            if (relativePath.startsWith('templates/components/')) {
-                components.push(relativePath);
-                continue;
-            }
+            // if (relativePath.startsWith('templates/components/')) {
+            //     components.push(relativePath);
+            //     continue;
+            // }
             if (relativePath.startsWith('helpers/')) {
                 helpers.push(relativePath);
                 continue;
