diff --git a/-private/environment/index.js b/-private/environment/index.js
index 217583f4bae53f6f72db8d3debd0f31b3dac2b8e..2bc41ac386aeaf66057edb6d65c1838c702a8075 100644
--- a/-private/environment/index.js
+++ b/-private/environment/index.js
@@ -96,10 +96,12 @@ function emberLooseEnvironment(options) {
                     // Classic controller
                     candidates.push(colocatedTemplatePath.replace('/controllers/', '/templates/'));
                 }
-                else if (colocatedTemplatePath.includes('/components/')) {
-                    // Classic layout component
-                    candidates.push(colocatedTemplatePath.replace('/components/', '/templates/components/'));
-                }
+                // we no longer need this as it remnant of classic components and it interferes with our route stricture in showcase up
+                // upstream PR: https://github.com/typed-ember/glint/pull/721
+                // else if (colocatedTemplatePath.includes('/components/')) {
+                //     // Classic layout component
+                //     candidates.push(colocatedTemplatePath.replace('/components/', '/templates/components/'));
+                // }
                 return candidates;
             },
         },
