diff --git a/packages/components/src/components/hds/advanced-table/column-manager/index.gts b/packages/components/src/components/hds/advanced-table/column-manager/index.gts
index 74218ec20f..d79037bc93 100644
--- a/packages/components/src/components/hds/advanced-table/column-manager/index.gts
+++ b/packages/components/src/components/hds/advanced-table/column-manager/index.gts
@@ -46,6 +46,9 @@ export interface HdsAdvancedTableColumnManagerSignature {
         columnOrder: HdsAdvancedTableSignature['Args']['columnOrder'];
         draggedColumnKey: HdsAdvancedTableNormalizedColumn['key'] | null;
         firstColumnKey: HdsAdvancedTableNormalizedColumn['key'] | undefined;
+        firstNonStickyColumnKey:
+          | HdsAdvancedTableNormalizedColumn['key']
+          | undefined;
         gridTemplateColumns: string;
         lastColumnKey: HdsAdvancedTableNormalizedColumn['key'] | undefined;
         orderedColumns: HdsAdvancedTableNormalizedColumn[];
@@ -166,6 +169,7 @@ export default class HdsAdvancedTableColumnManager extends Component<HdsAdvanced
             columnOrder=Order.columnOrder
             draggedColumnKey=Order.draggedColumnKey
             firstColumnKey=Order.firstColumnKey
+            firstNonStickyColumnKey=Order.firstNonStickyColumnKey
             gridTemplateColumns=Width.gridTemplateColumns
             lastColumnKey=Order.lastColumnKey
             orderedColumns=Order.orderedColumns
diff --git a/packages/components/src/components/hds/advanced-table/column-manager/order.gts b/packages/components/src/components/hds/advanced-table/column-manager/order.gts
index 4a4a307bda..7df9025966 100644
--- a/packages/components/src/components/hds/advanced-table/column-manager/order.gts
+++ b/packages/components/src/components/hds/advanced-table/column-manager/order.gts
@@ -42,6 +42,7 @@ interface HdsAdvancedTableColumnManagerOrderSignature {
         draggedColumnKey: string | null;
         reorderHoveredColumnKey: string | null;
         firstColumnKey: string | undefined;
+        firstNonStickyColumnKey: string | undefined;
         lastColumnKey: string | undefined;
         syncColumnOrder: ModifierLike<HdsAdvancedTableSyncColumnOrderSignature>;
         moveColumnToDropTarget: (
@@ -344,6 +345,7 @@ export default class HdsAdvancedTableColumnManagerOrder extends Component<HdsAdv
         draggedColumnKey=this.draggedColumnKey
         reorderHoveredColumnKey=this.reorderHoveredColumnKey
         firstColumnKey=this.firstColumnKey
+        firstNonStickyColumnKey=this.firstNonStickyColumnKey
         lastColumnKey=this.lastColumnKey
         syncColumnOrder=this.syncColumnOrder
         moveColumnToDropTarget=this.moveColumnToDropTarget
diff --git a/packages/components/src/components/hds/advanced-table/index.gts b/packages/components/src/components/hds/advanced-table/index.gts
index 5b8b35b2f7..6e9052eac8 100644
--- a/packages/components/src/components/hds/advanced-table/index.gts
+++ b/packages/components/src/components/hds/advanced-table/index.gts
@@ -995,6 +995,7 @@ export default class HdsAdvancedTable<
                       @compositeItem={{C.item}}
                       @draggedColumnKey={{CM.draggedColumnKey}}
                       @firstColumnKey={{CM.firstColumnKey}}
+                      @firstNonStickyColumnKey={{CM.firstNonStickyColumnKey}}
                       @hasReorderableColumns={{@hasReorderableColumns}}
                       @hasResizableColumns={{@hasResizableColumns}}
                       @hasSelectableRows={{this.isSelectable}}
diff --git a/packages/components/src/components/hds/advanced-table/th-context-menu.gts b/packages/components/src/components/hds/advanced-table/th-context-menu.gts
index aa24a409d9..0c4a33bba6 100644
--- a/packages/components/src/components/hds/advanced-table/th-context-menu.gts
+++ b/packages/components/src/components/hds/advanced-table/th-context-menu.gts
@@ -35,6 +35,7 @@ export interface HdsAdvancedTableThContextMenuSignature {
   Args: {
     column: HdsAdvancedTableNormalizedColumn;
     isFirstColumn: boolean;
+    isFirstNonStickyColumn: boolean;
     isLastColumn: boolean;
     hasResizableColumns?: boolean;
     hasReorderableColumns?: boolean;
@@ -100,7 +101,7 @@ export default class HdsAdvancedTableThContextMenu extends Component<HdsAdvanced
   }
 
   get _reorderOptions(): HdsAdvancedTableThContextMenuOption[] {
-    const { isFirstColumn, isLastColumn } = this.args;
+    const { isFirstNonStickyColumn, isLastColumn } = this.args;
 
     const translatedMoveColumnLabel = this.hdsIntl.t(
       'hds.advanced-table.th-context-menu.move-column',
@@ -116,7 +117,7 @@ export default class HdsAdvancedTableThContextMenu extends Component<HdsAdvanced
       },
     ];
 
-    if (!isFirstColumn) {
+    if (!isFirstNonStickyColumn) {
       const translatedMoveColumnToStartLabel = this.hdsIntl.t(
         'hds.advanced-table.th-context-menu.move-column-to-start',
         { default: 'Move column to start' }
diff --git a/packages/components/src/components/hds/advanced-table/th.gts b/packages/components/src/components/hds/advanced-table/th.gts
index e1b7818a6e..888078a65b 100644
--- a/packages/components/src/components/hds/advanced-table/th.gts
+++ b/packages/components/src/components/hds/advanced-table/th.gts
@@ -67,6 +67,7 @@ export interface HdsAdvancedTableThSignature {
     depth?: number;
     draggedColumnKey?: HdsAdvancedTableNormalizedColumn['key'] | null;
     firstColumnKey?: HdsAdvancedTableNormalizedColumn['key'];
+    firstNonStickyColumnKey?: HdsAdvancedTableNormalizedColumn['key'];
     hasExpandAllButton?: boolean;
     hasReorderableColumns?: HdsAdvancedTableSignature['Args']['hasReorderableColumns'];
     hasResizableColumns?: HdsAdvancedTableSignature['Args']['hasResizableColumns'];
@@ -185,6 +186,16 @@ export default class HdsAdvancedTableTh extends Component<HdsAdvancedTableThSign
     );
   }
 
+  get isFirstNonStickyColumn(): boolean {
+    const { column, firstNonStickyColumnKey } = this.args;
+
+    return (
+      firstNonStickyColumnKey !== undefined &&
+      column !== undefined &&
+      firstNonStickyColumnKey === column.key
+    );
+  }
+
   get isLastColumn(): boolean {
     const { column, lastColumnKey } = this.args;
 
@@ -500,6 +511,7 @@ export default class HdsAdvancedTableTh extends Component<HdsAdvancedTableThSign
               @hasReorderableColumns={{@hasReorderableColumns}}
               @hasResizableColumns={{@hasResizableColumns}}
               @isFirstColumn={{this.isFirstColumn}}
+              @isFirstNonStickyColumn={{this.isFirstNonStickyColumn}}
               @isLastColumn={{this.isLastColumn}}
               @isStickyColumn={{@isStickyColumn}}
               @reorderHandleElement={{this._reorderHandleElement}}
